README.rst

```diff
@@ -188,6 +193,41 @@ You need to put `rest_tornado` in the configuration file.
 
 Please be aware that depending on the salt-api implementation you use, saltpad may requires a different version of salt on the master side, please refer to the table at the beginning of the Readme.
 
+Rest_cherrypy single-app page deployment
+----------------------------------------
+
+The rest_cherrypy flavour allow the deployment of single-app pages directly by the salt-api. The salt-api needs to be configured this way:
```

Comment:
The rest_cherrypy flavour allow**s** the deployment of single-app pages directly by the salt-api. The salt-api needs to be configured this way: